.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "wcwave_adaptors"]
-	path = wcwave_adaptors
-	url = https://github.com/tri-state-epscor/wcwave_adaptors.git
```

Comment:
I added wcwave_adaptors module in https://github.com/tri-state-epscor/wcwave_adaptors into vwplatform module. I was encountering errors while inserting files into the virtual watershed with wcwave_adaptors submodule which was already in the vwplatform.